src/tutorials/first_game_br.md

```diff
@@ -0,0 +1,166 @@
+
+<!--- TODO image of properties window --->
+
+### Change Parachute Settings
+
+In the same **ServerContext** folder as the **SkydivePlayerSettings**, select the **ParachutePlayerSettings**. In the **Properties** window, change the **Ground > Max Speed** property to `2400`. This will increase the max speed moving forward while parachuting.
+
+<!--- TODO image of properties window --->
+
+## Changing the Play Zone
+
+The "Play Zone" is a cylindrical area where the player will *not* receive constant damage. The Play Zone will shrink in size for a certain amount of intervals.
+
+### Change the Play Zone Settings
+
+Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone** and open the **Properties** window. The object has many **Custom Properties** that will be described below:
+
+| Property Name | Description |
+| --- | --- |
+| Phase Count | The number of phases. The Play Area will shrink in size as the phases increase. It will stop shrinking once the last phase occurs. |
+| ZoneSizeRatio | The ratio for the Play Area to shrink after each phase. For example, `0.7` signifies the area will shrink down to 70% of its current phase size. |
+| StaticTime | The amount of seconds each phase will last before it starts shrinking. |
+| Closing Time | The amount of seconds it takes to shrink before starting the next phase. |
+| BaseDamageRate | The amount of damage per second applied to the players outside the Play Zone during phase 1. |
+| DamageMultiplier | The amount multiplied to BaseDamageRate after each phase. |
+| UseRoundTiming | The toggle to restart the Play Zone after each round. |
+| ActivationDelay | The amount of seconds until the first phase begins. |
+
+### Change the Initial Play Zone Size
+
+1. Navigate to **Gameplay Settings > Zone Settings > Constricting Play Zone > ServerContext**.
+2. Select the **InitialZone** object.
+3. Focus on the cylinder mesh by pressing ++F++.
+4. Press ++R++ to enter scale mode.
+5. Change the X and Y scale so the initial Play Zone is a different size.
+
+<!--- TODO image of smaller play zone --->
+
+## Changing the Loot Tables
+
+There are several LootSpawner object placed throughout the map that randomly roll a weapon from the Loot Table list. Each roll on the Loot Table has an equal chance to be rolled.
+
+### Add a New Roll to the Loot Table
+
+Navigate to **Gameplay Settings > Basic Loot Table > ServerContext > Loot** group and notice there are three objects inside representing the three different weapons in the Loot Table. Select one of the **Object (server)** and duplicate it using the keyboard shortcut ++Ctrl++ + ++W++.
+
+<!--- TODO image of hierarchy for new object --->
+
+### Change Weapon in the Loot Table
+
```

Comment:
Link to the weapons ref for if you want to learn to create entirely new custom weapons.